sig-cluster-lifecycle/charter.md

```diff
@@ -22,9 +22,7 @@ The following topics fall under ownership of this SIG:
 
 #### Cross-cutting and Externally Facing Processes
 
-- This SIG works closely with SIG Release during the end of the cycle, because of how the release of kubeadm is currently managed. The process requires a lock-step coordination which is outlined [here](https://github.com/kubernetes/kubeadm/blob/master/docs/release-cycle.md).
 - The SIG recommends and verifies compatibility of critical cluster add-ons for networking, network policy, service discovery, etc. The SIG maintains the health check of container images for some add-ons that are required for cluster bootstrapping. While the SIG could provide support to users that have add-on related issues, the SIG can decide to delegate issues to the add-on maintainers or other SIGs.
-- The SIG owns the framework for upgrade / downgrade testing of Kubernetes. However, the SIG does not own the individual tests for each feature. While the SIG is often involved in triaging and debugging upgrade or downgrade test failures, the SIG will normally delegate issues to the feature owner for the failing test.
```

Comment:
[1]

Reply:
> Agree on not owning the framework, but I wonder if this leaves the project without ownership of verifying pre-conditions / post-conditions that are expected to hold for kubernetes across cluster providers

my take about the upgrade framework is the following, the variety of Kubernetes component topologies and its modularity renders the chances of a single upgrade framework to work across cluster providers unlikely, unless we start standardizing topology.

the entry that is being removed here is something that works for [GCE/GKE](https://github.com/kubernetes/kubernetes/blob/master/test/e2e/cloud/gcp/cluster_upgrade.go#L614-L617), yet i cannot confirm if it currently supports pre/post condition testing across cluster providers.

> Put another way: there are some upgrade/skew tests in release-informing based on kind. Are those representative of how we would expect all kubernetes upgrades/skews to perform?

i'm surprised that kubeadm/kinder jobs are the only upgrade jobs in `release-*` dashboards at this point. the kubeadm upgrades do not strictly follow the upgrade/skew document, due to topology differences. but they are close on the critical parts.

next to cluster-api upgrades to be added as release-* signal, in my opinion cloud providers need to step up and start doing federated upgrade testing at minimum.